README.md

```diff
@@ -1 +1,117 @@
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 Awesome Books: Plain JavaScript with Objects <a name="about-project"></a>
+
+> This a to-do list web app where users can add, edit, and delete thier tasks. It built using plain JavaScript with ES6 and webpack. The aim of this project is to master ES6 and webpack. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> - HTML5
+> - CSS3
+> - JavaScript ES6
+> - Webpack
+
+### Key Features <a name="key-features"></a>
+
+> The key features of this website include.
+
+> - **Interactive**
+> - **Responsive**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to install npm and node.js as well as webpack and linter modules
+
+### Setup
+
+Clone this repository to your desired folder:
+1. Use VScode: open terminal in VScode
+2. Navigate to the directory where you want clone the copy of this repository
+3. Create new directory [optional]
+4. Clone the repo using the below command
+ > git clone  https://github.com/Microverse-Fullstack-Program/Webpack-Todo-Lists
+
+### Usage
+
+This project can be used to practice javascript ES6 and Webpack
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+> The collaborators of this project.
+
+👤 **Chere Lemma**
+
+- GitHub: [@githubhandle](https://github.com/cherelemma)
+- Twitter: [@twitterhandle](https://twitter.com/Chere21271613)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/chere-lemma27211613/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Microverse-Fullstack-Program/Webpack-Todo-Lists/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank [Microverseinc](https://github.com/microverseinc) for.
+
+* [Readme-template](https://github.com/microverseinc/readme-template)
+* [Linter for HTML and CSS ](https://github.com/microverseinc/linters-config)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] It's great that you have added a descriptive readme but i guess this link is not working 😨 It's because you don't hav e a license file in your repository 🤔 It would be nice if you add a license file using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 👍